test/onnx/test_fx_to_onnx.py

```diff
@@ -386,21 +386,14 @@ def forward(self, x):
                     fake_model, fake_x, export_options=export_options
                 )
 
-            # Scenario 2: Fake model and real input WITHOUT fake_context
-            with self.assertRaises(torch.onnx.OnnxExporterError):
```

Comment:
This scenario relys the error from `to_model_proto` crashing with fake tensor initializer which does not happen anymore after the filtering logic.

Reply:
How is it affected by this PR. weight and bias of `self.linear` should be parameter and hence still fake tensor right?

Reply:
Because previous to this PR, in this case, we don't actiavte fake_mode during the export, which triggers `to_model_proto` takes `include_initializers=True`, so we send FakeTensor into `_export_onnx`. 

But now, with the fix, this will never be the case. If we really want to error this case out, we could add Assertion into `_assert_fake_mode` for consistency, but for now, this case is actually supported.

Reply:
Discussed offline, to be brought back.